WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -194,6 +195,20 @@ final class EditableOrderViewModel: ObservableObject {
     ///
     @Published var selectedProductViewModel: ProductInOrderViewModel? = nil
 
+    /// Keeps track of selected/unselected Products, if any
+    ///
+    @Published var selectedProducts: [Product?] = []
```

Comment:
Are these properties necessary, or can they be taken from the `addProductViewModel?

Reply:
That's a good question 🤔 , at the moment these are necessary as in the `addProductViewModel` we just keep track of IDs, and we need to pass the model objects to add or remove `Products` and `ProductVariations`, but I'll reevaluate based on the direction we take with how we generate the `ProductSelectorViewModel`